openg2p_registration/models/registration_service.py

```diff
@@ -141,6 +209,12 @@ def create_disbursement_fields(self, rid, temp, regd):
                         data["bank_account_id"] = res.id
                 elif k == "phone":
                     data["phone"] = odk_data["phone"]
+                elif k == "bban":
+                    data["kyc_id"] = odk_data["bban"]
+                elif k == "new_emis_code":
+                    data["external_id"] = odk_data["new_emis_code"]
+                elif k == "town_village":
+                    data["town_village"] = odk_data["town_village"]
```

Comment:
What does town village have to do with auto deduplication?

Reply:
Its optional I will remove it.